var/spack/repos/builtin/packages/libspatialite/package.py

```diff
@@ -18,6 +18,7 @@ class Libspatialite(AutotoolsPackage):
 
```

Comment:
manual_download is only true for one version, correct? I would remove this

Reply:
Still need to remove `manual_download`

Reply:
oops. totally missed it. fixed now.